File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, arr: List[int], k: int) -> int:
        nums = set()

        for i in range(len(arr)):
            nums.add(arr[i])

        i = 1
        missing = -1

        while True:
            missing = i * k

            if missing not in nums:
                break

            i += 1

        return missing

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.js
/**
 * @param {number[]} arr
 * @param {number} k
 * @return {number}
 */
var missingMultiple = function(arr, k) {
    let nums = new Set();

    for(let i=0; i<arr.length; i++) {
        nums.add(arr[i]);
    }

    let i = 1;
    let missing = -1;

    while(true) {
        if(!nums.has(k * i)) {
            missing = k * i;
            break;
        }

        i++;
    }

    return missing;
};